### PR TITLE
Finaldifftool

### PR DESCRIPTION
**Pull request by lawrence-lee**
@gamerson 

The output isnt as pretty as before, but its completely legible just has one extra [] and extra , everywhere.

sample [diff](https://gist.github.com/lawrence-lee/1b74810960fb52d73756a78a2d7022fe)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -77,35 +77,37 @@ repositories {
...
 
+	[bndtoolsOutputFiles, gradleOutputFiles, mavenOutputFiles].each { replicationFiles ->
 		replicationFiles.each { replicationFile ->
 
 			def liferayGradleOutputFile = liferayGradleOutputFiles.find {
 				it.name == replicationFile.name
 			}
 
-			String exclusions = "Archiver-Version, Javac-Debug, Javac-Deprecation, Javac-Encoding, Built-By, Build-Jdk, *pom.properties, *pom.xml"
+			String exclusions = "*pom.properties, *pom.xml, Archiver-Version, Built-By, Build-Jdk, Javac-Debug, Javac-Deprecation, Javac-Encoding"
```
Comment (gamerson) [inline build.gradle]: This looks good, so lets add back the Archiver-Version an dthen I"ll merge it.

**Comment by gamerson** [comment]
Closing see comment, thanks!

**Comment by lawrence-lee** [comment]
Hey @gamerson, the exclusions are all there. I just alphabetized them with special chars first. Should I change the ordering?

**Comment by gamerson** [comment]
Oh i see, nevermind then! this looks good.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Sent https://github.com/gamerson/liferay-blade-samples/pull/35

**Pull request closed without merging**